clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -501,13 +506,16 @@ public StoreIngestionTask(
     if (!this.recordLevelMetricEnabled.get()) {
       LOGGER.info("Disabled record-level metric when ingesting current version: {}", kafkaVersionTopic);
     }
+    this.parallelProcessingThreadPool = builder.getAaWCWorkLoadProcessingThreadPool();
```

Comment:
nit: Should we use AAWC instead of AaWC as it is the pattern everywhere else.

Reply:
Sure